crates/cli/src/utils/mod.rs

```diff
@@ -78,9 +79,43 @@ pub fn subscriber() {
         .init()
 }
 
+#[derive(Debug, Clone)]
+pub enum ParseU256Error {
+    HexWithPrefix,
+    HexWithoutPrefix,
+    Decimal,
+}
```

Comment:
please use thiserror attributes instead of manually implementing display.

we also want to keep the actual error message